<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TTest11_8AClass</class>
 <widget class="QMainWindow" name="TTest11_8AClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TTest11_8A</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>周期:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spbPeriod"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>粒子:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spbGrain"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>振幅:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spbRadius"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>238</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Sin" name="mySin">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Sin</class>
   <extends>QFrame</extends>
   <header>Sin.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="TTest11_8A.qrc"/>
 </resources>
 <connections/>
</ui>
